lib/puppet/transaction/report.rb

```diff
@@ -377,7 +377,13 @@ def summary
   # @api public
   #
   def raw_summary
-    report = { "version" => { "config" => configuration_version, "puppet" => Puppet.version  } }
+    report = {
+      "version" => {
+        "config" => configuration_version,
+        "environment" => environment,
```

Comment:
I put `environment` inside the `version` key, as `#summary` expects all top level keys to be hashes (might be a bug?)

Reply:
Makes sense. Ideally we should have a schema for the last run *summary*, but I don't think we do?